tests/python/pants_test/base/test_exception_sink_integration.py

```diff
@@ -57,25 +46,31 @@ def _get_log_file_paths(self, workdir, pants_run):
 
   def test_logs_unhandled_exception(self):
     with temporary_dir() as tmpdir:
-      pants_run = self.run_pants_with_workdir([
-        # The backtrace should be omitted when printed to stderr, but not when logged.
-        '--no-print-exception-stacktrace',
-        'list',
-        '//:this-target-does-not-exist',
-      ], workdir=tmpdir)
+      pants_run = self.run_pants_with_workdir(
+        ['list', '//:this-target-does-not-exist'],
+        workdir=tmpdir,
+        # The backtrace should be omitted when --print-exception-stacktrace=False.
+        print_exception_stacktrace=False)
       self.assert_failure(pants_run)
-      self.assertIn('\n(backtrace omitted)\n', pants_run.stderr_data)
-      self.assertIn(' ResolveError: "this-target-does-not-exist" was not found in namespace "". Did you mean one of:',
-                    pants_run.stderr_data)
+      self.assertRegexpMatches(pants_run.stderr_data, """\\A\
+timestamp: ([^\n]+)
+Exception caught: \\(<class 'pants\\.build_graph\\.address_lookup_error\\.AddressLookupError'>\\)
```

Comment:
It'd be nice if the error could omit the `<class '`, `'>` bits and look more like `Exception caught: (pants.build_graph.address_lookup_error.AddressLookupError) (backtrace omitted)`.

Reply:
Oooooh, I totally agree with this one.

Reply:
Done!